massgen/orchestrator.py

```
@@ -3345,6 +3381,24 @@ async def _coordinate_agents(
             },
         )
 
+        # Clean up orphaned massgen/* branches from previous crashed sessions
+        coordination_config = getattr(self.config, "coordination_config", None)
+        _wm = getattr(coordination_config, "write_mode", None) if coordination_config else None
+        if _wm and _wm != "legacy":
+            from .filesystem_manager import IsolationContextManager
+
+            for agent_id_cleanup, agent_cleanup in self.agents.items():
```

Ruff (0.15.0)

[warning] Loop control variable `agent_id_cleanup` not used within loop body

Rename unused `agent_id_cleanup` to `_agent_id_cleanup`

(B007)